string color
---
bool is_success